node/node.go

```diff
@@ -999,7 +999,7 @@ func (n *Node) commonTransactionValidation(tx *transaction.Transaction, bypassSi
 	enableSignWithTxHash := currentEpoch >= n.enableSignTxWithHashEpoch
 
 	txSingleSigner := n.txSingleSigner
-	if bypassSignature {
+	if checkSignature {
```

Comment:
`if !checkSignature`

Reply:
good catch.

Reply:
That's why I said it is misleading. Should definitely be renamed to `bypassSignatureCheck`